### PR TITLE
update the document that setting sysctl

### LINKED ISSUE #40201: Document that setting sysctl will do nothing for Windows nodes
**This is a Feature Request**

**What would you like to be added**

On this page https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/ tell that sysctl do nothing on Windows.
/sig node

Context: https://kubernetes.slack.com/archives/C0BP8PW9G/p1679439424451039

**Why is this needed**

Document limitation and make sure there is no confusion.

### PR DESCRIPTION
**Pull request by AmarNathChary**
update the document that setting sysctl by adding (The node must be running Linux)
This PR Fixes #40201

### CODE REVIEW

File: content/en/includes/task-tutorial-prereqs.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-You need to have a Kubernetes cluster, and the kubectl command-line tool must
-be configured to communicate with your cluster. It is recommended to run this tutorial on a cluster with at least two nodes that are not acting as control plane hosts. If you do not already have a
+The node must be running Linux. you need to have a Kubernetes cluster, and the kubectl command-line
```
Comment (divya-mohan0209) [inline content/en/includes/task-tutorial-prereqs.md:1]:
```suggestion
You need to have a Kubernetes cluster, and the kubectl command-line
```

Comment (divya-mohan0209) [reply content/en/includes/task-tutorial-prereqs.md:1]: A more descriptive statement in the form of a note before this statement would make more sense. As it stands currently, there is no mention of sysctl anywhere.  Merely adding the statement you have does not help clarify that sysctl doesn't do anything for Windows nodes. WDYT?

Comment (AmarNathChary) [reply content/en/includes/task-tutorial-prereqs.md:1]: Hi @divya-mohan0209 thanks for suggesting I agree that keeping this statement more descriptive in the form of a Note. will look more precise for a reader. I will add those changes

**Comment by AmarNathChary** [comment]
Can anyone PTAL! at this PR

**Comment by divya-mohan0209** [comment]
Hello @AmarNathChary , please note that, per process, the PR will need to be technically and qualitatively reviewed. I have cc'ed the respective parties above for technical review.

File: content/en/docs/tasks/administer-cluster/sysctl-cluster.md [hunk 2]
```diff
@@ -26,6 +26,10 @@ the Linux man-pages project.
 {{< /note >}}
 ## {{% heading "prerequisites" %}}
 
+{{< note >}}
+Sysctl is a Linux-sepcific command-line tool used to configure various kernal parameters
+and it is not available on non-Linux operating systems
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/sysctl-cluster.md:31]:
```suggestion
and it is not available on non-Linux operating systems.
```

File: content/en/docs/tasks/administer-cluster/sysctl-cluster.md [hunk 3]
```diff
@@ -26,6 +26,10 @@ the Linux man-pages project.
 {{< /note >}}
 ## {{% heading "prerequisites" %}}
 
+{{< note >}}
+Sysctl is a Linux-sepcific command-line tool used to configure various kernal parameters
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/sysctl-cluster.md:30]:
```suggestion
`sysctl` is a Linux-sepcific command-line tool used to configure various kernal parameters
```

**Comment by AmarNathChary** [comment]
Hi @tengqm thanks for those corrections. committed those changes. PTAL!

File: content/en/docs/tasks/administer-cluster/sysctl-cluster.md [hunk 4]
```diff
@@ -26,6 +26,10 @@ the Linux man-pages project.
 {{< /note >}}
 ## {{% heading "prerequisites" %}}
 
+{{< note >}}
+`sysctl` is a Linux-specific command-line tool used to configure various kernel parameters
```
Comment (divya-mohan0209) [inline content/en/docs/tasks/administer-cluster/sysctl-cluster.md:30]: Per [this comment](https://github.com/kubernetes/website/issues/40201#issuecomment-1479868095), it'd be worthwhile to add that the parameters can be set for a pod running on Windows but will do nothing.

**Pull request merged**